files/en-us/tools/working_with_iframes/index.html

```diff
@@ -10,14 +10,26 @@
 ---
 <div>{{ToolsSidebar}}</div>
 
-<p>You can point the developer tools at a specific <a href="/en-US/docs/Web/HTML/Element/iframe"><code>iframe</code></a> within a document.</p>
+<p>You can point the developer tools at a specific <a href="/en-US/docs/Web/HTML/Element/iframe"><code>iframe</code></a> within a document. The <a href="/en-US/docs/Tools/Page_Inspector">Inspector</a>, <a href="/en-US/docs/Tools/Web_Console">Console</a>, <a href="/en-US/docs/Tools/Debugger">Debugger</a> and all other developer tools will then target that iframe (essentially behaving as if the rest of the page does not exist).</p>
 
 <p>{{EmbedYouTube("Me9hjqd74m8")}}</p>
 
-<p>You'll see a button in the toolbar:</p>
+<p>To set an iframe as the target for the developer tools, first select the <em>iframe button</em> to launch a popup listing all the iframes in the document (and the main document itself). Then Select an entry to make it the <em>target iframe</em>. Note that the iframe button is not displayed if the page doesn't have any iframes!</p>
 
-<p><img alt="" src="https://mdn.mozillademos.org/files/8547/frame-selection-button.png" style="display: block; margin-left: auto; margin-right: auto;">Click it, and you'll see a popup listing all the iframes in the document, as well as the main document itself.</p>
+<p>To set an iframe as the target for the developer tools:
+  
+  <ul>
+    <li>Select the <em>iframe context picker button</em> to launch a popup listing all the iframes in the document (and the main document itself). Note that the button is only displayed if the page doesn't have any iframes!</li>
+    <li>Select an entry to make it the <em>target iframe</em>.</li>
+  </ul>
+</p>
 
-<p><img alt="" src="https://mdn.mozillademos.org/files/8549/frame-selection-popup.png" style="display: block; margin-left: auto; margin-right: auto;"></p>
+<p><img alt="Screenshot showing how to set an iframe as the target of developer tools (using the iframe button)" src="developer_tools_select_iframe.png" /></p>
+
+<div class="notecard note">
+  <h4>Note</h4>
+  <p>The iframe context picker button feature is enabled by default (if it has been disabled the iframe button is never displayed). The feature can be re-enabled from the Settings, using checkbox shown below:</p>
```

Comment:
```suggestion
  <p>The iframe context picker button feature is enabled by default (if it has been disabled the iframe button is never displayed). The feature can be re-enabled from the Settings menu, using checkbox shown below:</p>
```
…or just dismiss this suggestion if *Settings* without *menu* is actually fine